src/settings/index.js

```diff
@@ -119,7 +120,7 @@ class Organization extends React.Component {
     // But for now ...
     const sections = this.sections.map(section => ({
       label: section.label,
-      pages: section.pages.filter(page => !page.iface || this.props.stripes.hasInterface(page.iface)),
+      pages: section.pages.filter(page => (!page.iface || this.props.stripes.hasInterface(page.iface)) && !page.disabled),
```

Comment:
Hmmmmm, seeing the `iface` filter here, could you just use that?

Reply:
I agree that should probably do the trick and hopefully simplify this workflow. I'll give it a try!

Reply:
Just needed to set the `.iface` property to `login-saml` and the rest is handled properly. Thanks for the tip!